src/hooks/useThemeInfo.ts

```diff
@@ -0,0 +1,6 @@
+import useApiRequest from './useApiRequest';
+import { fetchThemeInfo } from '@/api/ThemeListApi';
+
+export default function useThemeInfo(themeId: number) {
+  return useApiRequest(() => fetchThemeInfo(themeId), [themeId]);
```

Comment:
```suggestion
  return useApiRequest(fetchThemeInfo, [themeId]);
```
useApiRequest에서 함수에 args를 넘겨주는 방식으로 만들어주시면 좋답니다 ㅎㅎ

Reply:
```tsx
const useApiRequest = (request, args = [])
useEffect(() => {
  request(...args)
}, [args])
```

요런느낌..

Reply:
이렇게 하는 목적은, reuqest 함수가 지금은 렌더링이 될 때마다 매번 새로 만들어지고 있어요.
fetchThemeInfo를 그대로 request로 사용할 경우 똑같은 참조를 유지할 수 있답니다!

이게 어떤 문제를 해결해줄 수 있을까요?